CONTRIBUTING.rst

```diff
@@ -0,0 +1,77 @@
+********************************************************************
+Contributing Guide
+********************************************************************
+
+Welcome to the Tensile project! If you're thinking about contributing, this document is for you, and is intended to help newcomers get acquainted with the contribution workflow.
+
+.. tip::
+   
+   If you haven't already, please review :ref:`getting-started` for an introduction to the project. For details on environment setup and day-to-day development processes, please refer to the :ref:`developer-guide`.
+
+
+============================
+How to submit a Pull Request
+============================
+
+When making changes:
```

Comment:
Should we discuss PR scope?  That is, keeping scope narrow. Also, what about force pushing